### PR TITLE
[PoC] Add `linter.<id>.target` option into base

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

This is just a PoC.

> Refer related issues or pull requests (e.g. `Close #123` or `None`).

Follow-up of #2191

> Check the following if needed.

- [ ] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

File: lib/runners/processor/coffeelint.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Processor::Coffeelint < Processor
 
       # @type self: SchemaClass
       let :config, npm(
+        target: false,
         file: string?,
       )
```
Comment (ybiquitous) [inline lib/runners/processor/coffeelint.rb:12]: @yykamei What do you think about this code? I feel not intuitive so much.

**Comment by ybiquitous** [comment]
There are the following cases, so it seems hard to write code to handle well such cases... 🤔 

---

There are linters that:
- support multiple `target`
- support only single `target`
- don't support any `target`

File: lib/runners/processor/coffeelint.rb (hunk 1 above)
Comment (yykamei) [reply lib/runners/processor/coffeelint.rb:12]: Thank you. I think so.

**Comment by ybiquitous** [comment]
@yykamei Can I close this PR if you don't mind? Of course, you can try it again.

**Comment by yykamei** [comment]
Of course. Thanks for taking time for this :pray:

**Comment by ybiquitous** [comment]
Don't mind. 😄

**Pull request closed without merging**